lib/minify.js

```diff
@@ -157,13 +166,28 @@ function minify(files, options) {
             }
             options.parse = options.parse || {};
             options.parse.toplevel = null;
+            var source_map_content = options.sourceMap && options.sourceMap.content;
+            if (typeof source_map_content == "string" && source_map_content != "inline" && source_map_content != "auto") {
+                source_map_content = parse_source_map(source_map_content);
+            }
+            source_maps = source_map_content && Object.create(null);
```

Comment:
According to https://github.com/terser-js/terser/pull/300#issuecomment-469037773 I guess it would be much better to use smth like:
```javascript
if (source_map_content) source_maps = new Set();
```

Reply:
At least it will reduce variety in code which doing same things

Reply:
👍 , but it's actually a Map.